src/interfaceOfNetwork.ts

```diff
@@ -26,6 +26,10 @@ export interface ITransactionOnNetwork {
     receipt: ITransactionReceipt;
     contractResults: IContractResults;
     logs: ITransactionLogs;
+
+    // TODO: In a future major release, make these required (empty is allowed).
```

Comment:
Actually, maybe, we'll completely drop these interfaces, and use simple DTOs (according to the specs).